apps/server/src/models/user.ts

```diff
@@ -13,6 +14,22 @@ export class User extends TimeStamps {
 
     @prop()
     public providerId: string
+
+    public toJSON() {
+        return {
+            nickname: this.nickname,
+            provider: this.provider,
+            providerId: this.providerId,
+        }
```

Comment:
각 provider id 는 fe 에서 필요 없을거라 노출하지 않는게 좋을거 같아요
```suggestion
        return {
            _id: this._id,
            nickname: this.nickname,
            provider: this.provider,
        }
```